smart-contracts/data/sifnode_mainnet_update_24_Nov_2021.json

```diff
@@ -17,7 +17,8 @@
     ],
     "unit_denom": "",
     "ibc_counterparty_denom": "",
-    "ibc_counterparty_chain_id": ""
+    "ibc_counterparty_chain_id": "",
+    "is_whitelisted": true
```

Comment:
the is_whitelisted is not present on the protocol side anymore , just in case if that is of any relevence to this change 
http://github.com/Sifchain/sifnode/blob/279fe96e826b2da1218c5ac4c68b7e82182ce9a3/proto/sifnode/tokenregistry/v1/types.proto#L21-L54

Reply:
@mccallofthewild all entries on the list are now considered whitelisted so this property has been removed.